Makefile

```diff
@@ -6,8 +6,8 @@ v: v.c
 	./v -o v compiler
 	rm v.c
 
-v.c:
-	curl -Os https://raw.githubusercontent.com/vlang/vc/master/v.c
+v.c: Makefile
```

Comment:
Why this?

Reply:
Update v.c when the v.c URL in the Makefile is updated

Reply:
I get it now. Thanks for the explanation.

Reply:
But this is the simplest way.